meson.build

```diff
@@ -465,6 +413,46 @@ symb_defs = {
     'cblas_?dotc_sub':  {'def': ['CBLAS', 'FORCE_USE_STACK', 'CONJ']},
 }
 
+# config.h file generation
+
+run_command('c_check', 'Makefile.conf', './build/_config_1.h', 'gcc')
+run_command('f_check', 'Makefile.conf', './build/_config_1.h', 'gfortran')
```

Comment:
`./build/` shouldn't be hardcoded.

Reply:
Also, I was wondering about the order / when these are executed, is it always in the configure stage?

Reply:
The reason is that instead of `gcc` / `gfortran` we need to run it with `cc.get_id()`, also if it isn't `gcc` or `gfortran` there's a dependency on `ctest*.c` and `ftest*.f`

Reply:
Do you know how to retrieve build directory in `meson.build`? It looks like there's no way to access it, so I guess `./build/` should be hardcoded?

Reply:
AFAIK `build_dir` https://mesonbuild.com/Reference-manual_builtin_meson.html#mesoncurrent_build_dir

Reply:
Done!

> Also, I was wondering about the order / when these are executed, is it always in the configure stage?

I think that `run_command` is always executed in the setup/configure stage, and I think it always executes line by line.